modules/webhook-configuring-acs.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-using-generic-webhooks.adoc
+:_module-type: PROCEDURE
+[id="webhook-configuring-acs_{context}"]
+= Configuring {product-title}
+
+[role="_abstract"]
+Create a new integration in {product-title} by using the webhook URL.
+
+.Procedure
+. On the RHACS portal, navigate to *Platform Configuration* -> *Integrations*.
+. Scroll down to the *Notifier Integrations* section and select *PagerDuty*.
+. Click *New Integration* (add icon).
+. Enter a name for *Integration Name*.
+. Enter the webhook URL in the *Endpoint* field.
+. If your webhook receiver is using an untrusted certificate, enter a CA certificate in the *CA Cert* box, otherwise leave it blank.
++
+[NOTE]
+====
+The server certificate used by the webhook receiver must be valid for the endpoint DNS name.
+You can turn on the *Skip TLS Verify* toggle to ignore this validation.
+Red Hat does not recommend turning off TLS verification.
+====
+. Optionally, turn on the *Enable Audit Logging* toggle, to receive alerts about all the changes made in {product-title}.
```

Comment:
```suggestion
. Optional: Turn on the *Enable Audit Logging* toggle to receive alerts about all the changes made in {product-title}.
```
We've generally ascribed to the ISG example:
>  If a step is optional, begin the step with the word _Optional_. If your authoring tool provides a semantic element to generate the word _Optional_, use that element.
1. Optional: Click **Set password**. In the **Password** field, type a password.